plugins/woocommerce/client/blocks/tests/e2e/tests/mini-cart/mini-cart.block_theme.spec.ts

```diff
@@ -7,15 +7,15 @@ import { test, expect, BlockData } from '@woocommerce/e2e-utils';
  * Internal dependencies
  */
 import { REGULAR_PRICED_PRODUCT_NAME } from '../checkout/constants';
+import { blockData as baseBlockData } from './utils';
 
-const blockData: BlockData = {
-	name: 'Mini-Cart',
-	slug: 'woocommerce/mini-cart',
-	mainClass: '.wc-block-minicart',
-	selectors: {
-		frontend: {},
-		editor: {},
-	},
+type ExtendedBlockData = BlockData & {
+	productPage: string;
+};
+
+const blockData: ExtendedBlockData = {
+	...baseBlockData,
+	productPage: '/product/polo/',
```

Comment:
Rather than altering the `BlockData` type, I used an intersection type to add the specific `productPage` property needed for this context.